library/include/rocrand_mtgp32.h

```diff
@@ -128,9 +128,9 @@ typedef mtgp32_params_fast_t mtgp32_fast_params;
 
 struct mtgp32_state
 {
-    unsigned int status[MTGP_STATE];
     int offset;
     int id;
+    unsigned int status[MTGP_STATE];
```

Comment:
You are just changing the order. Is this the workaround?

Reply:
Yes.  Changing the order of variables seems to affect how the struct is packed.

Reply:
Rather silly, not sure if you filed a bug.

Reply:
Agreed, it seems related to the targetid changes, I'll file an internal ticket later.

Reply:
This is interesting.   Did you profile to see what functions were mostly affected by the struct alignment changes?

Reply:
The biggest performance changes were seen in the device kernel functions, specifically for the uniform-uint, uniform-double, and uniform-float types (close to 40%).  Normal-float and log-normal float had smaller performance hits (< 10%).